brave/brave-ios
# Fix ##6687: Don't show sms/tel dialog for non-active tabs

Adding active tab url check for tel / sms and changing the alert description



## Summary of Changes


This pull request fixes #6687

## Submitter Checklist:


## Test Plan:

Test 1: 

* Visit hxxps://csrf.jp/brave/sms.php
* Tap "Click Me" button
* [google.com](http://google.com/) is opened in the new tab
* Confirmation dialog for sms: link should not be shown on [google.com](http://google.com/)

Test 2: 

* Visit a website with a call to action like a business on [google](https://www.google.com/search?q=carpet&source=hp&ei=cOy1Y9qyEeXQkPIP64SFuAU&oq=carpet&gs_lcp=ChFtb2JpbGUtZ3dzLXdpei1ocBADMggIABCABBCxAzILCAAQgAQQsQMQgwEyBQgAEIAEMgUIABCABDILCC4QgAQQsQMQ1AIyBAgAEAMyCAgAEIAEEMkDMgUIABCSAzoOCAAQjwEQ6gIQjAMQ5QI6EQguEIAEELEDEIMBEMcBENEDOgsILhCABBCxAxCDAToICAAQsQMQgwE6DgguEIAEELEDEIMBENQCOggILhCABBDUAjoUCC4QgAQQsQMQgwEQxwEQ0QMQ1AI6DgguEIAEELEDEMcBENEDOgsIABCABBDJAxCLAzoICAAQkgMQiwM6DgguELEDEIMBEMcBENEDOgsILhCABBDHARDRAzoOCC4QgwEQ1AIQsQMQgAQ6EQguEIAEELEDEIMBEMcBEK8BOg4ILhCABBDJAxDHARCvAToICC4QgAQQsQM6CAguELEDEIMBUIkSWMhlYJlsaAhwAHgAgAGVAYgBowuSAQM0LjmYAQCgAQGwAQ-4AQI&sclient=mobile-gws-wiz-hp&dlnr=1&sei=hOy1Y-F-5KWq2w_mx6DIBQ#dlnr=1&trex=m_t:lcl_akp,rc_f:rln,rc_ludocids:15437203984229734589,ru_gwp:0%252C7,ru_lqi:CgZjYXJwZXQxfRUaAAAAAABIh87qoeeAgIAIWhIQABgAIgZjYXJwZXQqBAgDEACSAQ5mbG9vcmluZ19zdG9yZZoBJENoZERTVWhOTUc5blMwVkpRMEZuU1VSWE9YWkxSSFZSUlJBQqoBDhABKgoiBmNhcnBldCgA,ru_phdesc:M3Ayh4oddxI,trex_id:N7qfbb&lpg=cid:CgIgAQ%3D%3D) 

* Press call button
* Check new dialog includes website

## Screenshots:

![yuttree1-11_1](https://user-images.githubusercontent.com/6643505/211091341-083a7651-e18b-47ea-aec7-4c6078740c7f.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
